tests/integration/targets/inventory/runme.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+set -eux
+
+# TODO: remove env var and uncomment when ansibe provider is available 
+export TESTING_STATE_FILE="True"
+
+# terraform init
+# terraform apply -auto-approve
+
+ansible-playbook -i localhost, -i inventory1.yml test.yml
+
+# (
+#     mkdir -p subdir/
+#     cp test.yml subdir/
+#     cp ansible_provider.tf subdir/
+#     cd subdir/
+#     terraform init
+#     terraform apply -auto-approve -state mycustomstate.tfstate
+# )
+
+# ansible-playbook -i localhost, -i inventory2.yml test.yml
```

Comment:
I'd much rather you split these tests into separate targets with names. Otherwise, if a test with "inventory3.yml" fails, there is no direct indication of the part of the fuctionality that does not work.

Different targets would also make tests isolated, as right now, they all share `subdir/`, which means cross-contamination is possible, since the final test is always the same, and you can't be 100% sure that you're testing the correct thing.

Reply:
So the only reason to have inventory1-4.yml files is to test the options(cfg). There is only one test.yml since I only have one .tf file that generates .tfstate file, thus only one inventory. I could make more .tf files and different tests, but don't see the the benefit of this. To be 100% sure I created different subdir for each of the cfg file.